packages/api/cms-api/src/builds/changes-checker.console.ts

```diff
@@ -22,7 +19,7 @@ export class ChangesCheckerConsole {
 
         if (await this.buildsService.hasChangesSinceLastBuild()) {
             console.log("Changes detected, starting build...");
-            await this.buildsService.createBuilds("changesDetected");
+            await this.buildsService.dangerouslyCreateBuildsWithoutUserCheck("changesDetected");
```

Comment:
We're still planning to only start selected build based on the changes, am I right?

Reply:
absolutely